components/SiteNavbar.vue

```diff
@@ -1,56 +1,80 @@
 <template>
-  <nav class="border-b border-gray-300">
+  <div class="border-b border-gray-300">
     <div
       class="
         mx-auto
         container
-        flex flex-wrap flex-row-reverse
+        flex flex-wrap
         justify-between
         px-3
         py-3
         lg:flex-row
```

Comment:
You should add the `relative` class. Right now the absolute positioned buttons are positioned in relation to the max width of this element meaning it is offset to the right between the `md` and mobile breakpoints. Adding `relative` will position them with the containers margin.